app/assets/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -331,7 +331,7 @@ export function performDiffTracing(
   }
 }
 
-export function* saveTracingAsync(): Generator<*, *, *> {
+export function* saveTracingAsync(): Generator<any, any, any> {
```

Comment:
Hm, I guess there's not much we can do here, since the types which are passed into/out of the generator are so inhomogenous :(